frontend/pages/admin/IntegrationsPage/cards/AutomaticEnrollment/AutomaticEnrollment.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useContext } from "react";
+import { useQuery } from "react-query";
+import { AxiosError } from "axios";
+
+import { AppContext } from "context/app";
+import { IMdmApple } from "interfaces/mdm";
+import mdmAppleAPI from "services/entities/mdm_apple";
+
+import Spinner from "components/Spinner";
+import DataError from "components/DataError";
+import PremiumFeatureMessage from "components/PremiumFeatureMessage/PremiumFeatureMessage";
+import EmptyTable from "components/EmptyTable/EmptyTable";
+import Button from "components/buttons/Button/Button";
+import AppleBusinessManagerSection from "./components/AppleBusinessManagerSection/AppleBusinessManagerSection";
+import IdpSection from "./components/IdpSection/IdpSection";
+import EulaSection from "./components/EulaSection/EulaSection";
+
+const baseClass = "automatic-enrollment";
+
+const AutomaticEnrollment = () => {
+  const { config, isPremiumTier } = useContext(AppContext);
+
+  const { isLoading: isLoadingMdmApple, error: errorMdmApple } = useQuery<
+    IMdmApple,
+    AxiosError
+  >(["appleAPNInfo"], () => mdmAppleAPI.getAppleAPNInfo(), {
+    refetchOnWindowFocus: false,
+    retry: false,
+    enabled: config?.mdm.enabled_and_configured,
+  });
+
+  if (!isPremiumTier) return <PremiumFeatureMessage />;
+
+  if (isLoadingMdmApple) {
+    return <Spinner />;
+  }
+
+  if (errorMdmApple?.status === 404) {
+    return (
+      <EmptyTable
+        header="Automatic enrollment for macOS hosts"
+        info="Connect Fleet to the Apple Push Certificates Portal to get started."
+        primaryButton={<Button>Connect</Button>}
```

Comment:
What happens when this button is clicked?

Reply:
oh thanks, forgot to hook up its navigation. its goes the the integration mdm page. I'll add this.